curation-migration-datasync/database/dataService.ts

```diff
@@ -0,0 +1,160 @@
+import { Knex } from 'knex';
+import {
+  CuratedFeedItem,
+  CuratedFeedProspectItem,
+  CuratedFeedQueuedItems,
+  TileSource,
+} from '../types';
+import config from '../config';
+
+/**
+ * Exported only for test mocks
+ */
+export const queries = {
```

Comment:
this should be absorbed into the dataservice class

Reply:
done